starter-code/MyLinkedList.java

```diff
@@ -12,15 +12,46 @@ public int getSize() {
 	}
 
 	public boolean remove(int index) {
-		//to-do
+		if (index > size) {
+			return false;
+		} else {
+			Node<T> headNode = head;
+			Node<T> nextNode;
+			for (int i = 0; i < index; i++) {
+				headNode = headNode.getNext();
+			}
+			nextNode = headNode.getNext();
+			headNode.setNext(nextNode);
```

Comment:
I think you mean to be moving the next for `headNode` one forward, so it skips over the node that was previously next. However, this is actually just taking the same node that was next, and setting it to remain next.